src/parser.rs

```diff
@@ -1275,13 +1276,24 @@ impl<'a> Parser<'a> {
                 expr: Box::new(expr),
                 trim_where,
                 trim_what: Some(trim_what),
+                trim_characters: None,
+            })
+        } else if self.consume_token(&Token::Comma) && dialect_of!(self is SnowflakeDialect) {
```

Comment:
no `FROM` and `Snowflake` and next is `Comma` (by doc Snowflake doesn't support `FROM` syntax) -> `parse characters`